src/editors/TextEditor/ImageUpload/Wizard/ImageUploadModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalDialog, ActionRow, Button } from '@edx/paragon';
+
+const ImageUploadModal = ({ isOpen, close }) => (
+  <ModalDialog
+    title="My dialog"
+    isOpen={isOpen}
+    onClose={close}
+    size="lg"
+    variant="default"
+    hasCloseButton
+    isFullscreenOnMobile
+  >
+    <ModalDialog.Header>
+      <ModalDialog.Title>
+        Im a dialog box
+      </ModalDialog.Title>
+    </ModalDialog.Header>
+    <ModalDialog.Body>
+      <p>
+        Im baby palo santo ugh celiac fashion axe.
```

Comment:
This is just placeholder lorem ipsum for the modal.

Reply:
Nice placeholder text.